test/feature-benchmark/mzcompose.py

```diff
@@ -237,6 +253,13 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         help="'This' Materialize container tag to benchmark. If not provided, the current source will be used.",
     )
 
+    parser.add_argument(
+        "--this-balancerd",
```

Comment:
@nrainer-materialize I want to give the scalability framework such an option as well. Do you mind if I do that in a follow-up PR?

Reply:
Feel free to do that!